packet_analysis/sql/dao/arp.py

```diff
@@ -0,0 +1,24 @@
+from scapy.all import Packet
```

Comment:
Please utilize Pycharm's autoformat for this file

Reply:
Just gonna comment on this one since this is repeated on basically every file. I've changed some of my formatting preferences to remove the extraneous spaces that it loves to add to files. I'll reset those settings, but that is probably the only 'issue' these files have.